docs/configuration/executor.md

```diff
@@ -77,9 +77,9 @@ The default configuration for Helm chart can be found in the [values.yaml](https
 
 When executing a `kubectl` command, Botkube takes into account only bindings for a given execution Namespace. For example:
 
-- `@Botkube get po/botkube -n botkube` - collect `kubectl` executor bindings that include `botkube` or `*.` (all) Namespaces.
-- `@Botkube get po -A` - collect all `kubectl` executor bindings that include `*.` (all) Namespaces.
-- `@Botkube get po` - first, we resolve the execution Namespace. For that, we collect all enabled `kubectl` executor bindings and check the `defaultNamespace` property. If property is not define, we use the `default` Namespace. With resolved execution Namespace, we run the logic define in the first step.
+- `@Botkube kubectl get po/botkube -n botkube` - collect `kubectl` executor bindings that include `botkube` or `*.` (all) Namespaces.
+- `@Botkube kubectl get po -A` - collect all `kubectl` executor bindings that include `*.` (all) Namespaces.
+- `@Botkube kubectl get po` - first, we resolve the execution Namespace. For that, we collect all enabled `kubectl` executor bindings and check the `defaultNamespace` property. If property is not define, we use the `default` Namespace. With resolved execution Namespace, we run the logic define in the first step.
```

Comment:
From now on it will be required to have kubectl prefix?

Reply:
Yup, we're taking the deprecation notice to the next level, [see here](https://github.com/kubeshop/botkube/issues/802).

>In 0.15:
>
> - we change [the warning](https://botkube.io/docs/usage/#debugging-deployments-using-botkube) and say that k|kc|kubectl will be required in the 0.17 version
> - we replace all text examples and add required prefix e.g. change command from `@Botkube get po` to `@Botkube kubectl get po` in the Help message. 
>
>announce that in the release notes

Reply:
We have something similar in https://botkube.io/docs/usage/
Exactly this:
<img width="1005" alt="Screenshot 2022-10-18 at 10 08 47" src="https://user-images.githubusercontent.com/14016055/196388225-edebde10-1a23-42d8-93bd-975484f8d9c3.png">

WDYT to make it in the similar way?

Reply:
This has already been removed in @pkosiec latest doc re-organisation.

So no longer an issue 🙂 .. See https://botkube.io/docs/next/installation/

Reply:
Yup, I just found it and updated it below 👇 .